Source/MaestroThermal.cpp

```diff
@@ -159,7 +172,20 @@ void Maestro::MakeExplicitThermalHterm(Vector<MultiFab>& thermal,
     // turn off multigrid coarsening since no actual solve is performed
     info.setMaxCoarseningLevel(0);
 
-    MLABecLaplacian mlabec(geom, grids, dmap, info);
+    amrex::Vector<amrex::Geometry> small_geom(finest_level+1);
```

Comment:
we can do this to construct the small vectors, e.g.:

```
amrex::Vector<amrex::Geometry> small_geom(geom.begin(), geom.begin() + finest_level + 1);
```

Reply:
Maestro inherits a function from AmrCore, `Vector<Geometry> Geom (int a_coarsest_lev, int a_finest_lev) const`.

Reply:
ah, that's easy, thank you!